app/assets/stylesheets/modules/_body.scss

```diff
@@ -2,3 +2,36 @@
   padding: 3em 0;
   margin-top: 2em;
 }
+
+
+#chart-container {
+  #benchmark-types-form-show {
+    padding-bottom: 8px;
+    padding-top: 8px;
+  }
+
+  .alert-warning {
+    padding: 8px;
+  }
+}
+
+@media (max-width: 547px) {
```

Comment:
Could we use foundation's media queries here instead? http://foundation.zurb.com/docs/media-queries.html :smile_cat: